### PR TITLE
Ks sk937 manual install wp translation

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done / link checked with gitpod

I delete paragraphs about CloudDB for CA and APAC subs

Need proof for EN and Latin version. No DE versions because the structure of "cms manual installations" is different than the other subs ("imbrications").

### CODE REVIEW

File: pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md [hunk 1]
```diff
@@ -6,209 +6,300 @@ section: CMS
...
-Una vez completada la transferencia, asegúrese de que todos los archivos y carpetas se han copiado correctamente.
+>[!warning]
+>
+> Le recomendamos encarecidamente que utilice una carpeta raíz vacía para evitar conflictos con otro sitio web. Compruebe que la carpeta de destino no contenga ningún elemento antes de mover los archivos.
+>
 
-![wordpress](images/3131.png){.thumbnail}
+>[!primary]
+>
+Si el directorio raíz que ha definido no se ha creado automáticamente en las acciones descritas en el [etapa 1](#step1), puede crearlo a través de Filezilla.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md]:
```suggestion
> Si el directorio raíz que ha definido no se ha creado automáticamente en las acciones descritas en el [etapa 1](#step1), puede crearlo a través de Filezilla.

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md [hunk 2]
```diff
@@ -6,209 +6,300 @@ section: CMS
...
 
-También debe estar en posesión de su identificador y la contraseña de su base de datos SQL para conectarse a esta última. 
+## Procedimiento
 
+### Etapa 1 - preparar la instalación <a name="step1"></a>
 
-- Si necesita ayuda sobre cómo recuperar sus claves SQL, consulte la siguiente guía: []({legacy}1374)
+Para instalar el CMS **WordPress** en su plan [hosting de OVHcloud](https://www.ovhcloud.com/es-es/web-hosting/), es necesario realizar algunos preparativos.
 
+##### 1.1 - Comprobar la declaración de la "carpeta raíz"
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md]:
```suggestion
#### 1.1 - Comprobar la declaración de la "carpeta raíz"

```

Comment (OVH-Team-Guides) [reply pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md]: Please look on all subs for H4 subtitles which may have been changed to H5

File: pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md [hunk 3]
```diff
@@ -6,209 +6,300 @@ section: CMS
...
 
-Haga clic con el botón derecho sobre esta carpeta y seleccione «Extraer todo...». 
+- Obtenga [la información necesaria para conectarse al espacio FTP de su alojamiento web](https://docs.ovh.com/es/hosting/conexion-espacio-almacenamiento-ftp-alojamiento-web/#1-obtener-los-datos-de-conexion).
+- Consulte los accesos a la base de datos de su plan de hosting, si ya existe, o cree una con ayuda de [documentación](https://docs.ovh.com/es/hosting/crear-base-de-datos/).
 
-Indique un destino para extraer los archivos en una nueva carpeta. 
+#### 1.3 - Instalar el cliente FTP gratuito "Filezilla"
 
-En internet encontrará diversos tutoriales y programas de descompresión que le ayudarán a realizar esta operación. 
+Para más información, consulte la página de descarga gratuita y el tutorial sobre su uso en nuestra documentación sobre el [uso de Filezilla en el alojamiento de OVHcloud](https://docs.ovh.com/es/hosting/web_hosting_guia_de_uso_de_filezilla/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.es-es.md]:
```suggestion
Para más información, consulte la página de descarga gratuita y el tutorial sobre su uso en nuestra documentación sobre el [uso de FileZilla en el alojamiento de OVHcloud](https://docs.ovh.com/es/hosting/web_hosting_guia_de_uso_de_filezilla/).

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md [hunk 4]
```diff
@@ -1,220 +1,297 @@
...
+- Récupérez [les informations nécessaires pour vous connecter à l'espace FTP de votre hébergement web](https://docs.ovh.com/ca/fr/hosting/connexion-espace-stockage-ftp-hebergement-web/#etape-1-recuperer-les-informations-necessaires-pour-se-connecter).
+- Récupérez les accès à la base de données de votre offre d'hébergement web si elle existe déjà, ou créez-en une à l'aide de notre [documentation](https://docs.ovh.com/ca/fr/hosting/creer-base-de-donnees/).
 
-Dans la partie "Site local", qui correspond à la liste des fichiers présents sur votre ordinateur, ouvrez le dossier décompressé intitulé "wordpress" dans lequel sont présents les fichiers du CMS.
+#### 1.3 - Installer le client FTP gratuit « Filezilla »
 
-Dans la partie "Site distant" qui correspond dans ce cas à votre hébergement mutualisé OVHcloud, ouvrez le dossier "www". C'est dans ce dossier que tous les fichiers du CMS devront être déposés.
+Retrouvez le lien de téléchargement gratuit ainsi qu'un tutoriel sur son utilisation dans notre documentation sur l'[utilisation de Filezilla avec votre offre d'hébergement OVHcloud](https://docs.ovh.com/ca/fr/hosting/mutualise-guide-utilisation-filezilla/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md]:
```suggestion
Retrouvez le lien de téléchargement gratuit ainsi qu'un tutoriel sur son utilisation dans notre documentation sur l'[utilisation de FileZilla avec votre offre d'hébergement OVHcloud](https://docs.ovh.com/ca/fr/hosting/mutualise-guide-utilisation-filezilla/).

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md [hunk 5]
```diff
@@ -1,220 +1,297 @@
...
 
-*Si ce dossier n'existe pas il vous est possible de le créer.*
+#### 1.4 - Préparer une base de données <a name="step1-4"></a>
 
-*Vos fichiers doivent obligatoirement être déposés dans le dossier "www" sinon la procédure d'installation ne sera pas accessible depuis votre nom de domaine.*
+Les CMS ont besoin d'une base de données pour fonctionner. Nos offres d'[hébergement web](https://www.ovhcloud.com/fr-ca/web-hosting/) en contiennent, à l'execption de [l'hébergement gratuit Start 10M](https://www.ovhcloud.com/fr-ca/domains/free-web-hosting/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md]:
```suggestion
Les CMS ont besoin d'une base de données pour fonctionner. Nos offres d'[hébergement web](https://www.ovhcloud.com/fr-ca/web-hosting/) en contiennent, à l'exception de [l'hébergement gratuit Start 10M](https://www.ovhcloud.com/fr-ca/domains/free-web-hosting/).

```

Comment (OVH-Team-Guides) [reply pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md]: Start 10M is a Europe-only offer, it needs to be removed from CA/APAC subs.

File: pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md [hunk 6]
```diff
@@ -1,220 +1,297 @@
...
-Une fois le transfert terminé, assurez-vous que tous les fichiers et les dossiers ont été correctement transférés.
+Pour cela, ouvrez le dossier dans lequel vous avez téléchargé le fichier compressé, faites un clic-droit sur le fichier en question puis sélectionnez « Extraire tout... ».
 
-Cette opération conclut la partie consacrée au dépôt des fichiers sur le FTP.
+Indiquez le dossier « **WordPress** » en destination afin d'extraire vos fichiers dans ce dossier.
 
+#### 2.3 - Déplacer les fichiers sources du dossier « WordPress » vers le « dossier racine » sur votre hébergement web
 
-![hosting](images/3131.png){.thumbnail}
+Une fois les fichiers décompressés dans votre dossier « **WordPress** », [connectez-vous en FTP à votre espace de stockage](https://docs.ovh.com/ca/fr/hosting/connexion-espace-stockage-ftp-hebergement-web/) à l'aide du [client FTP Filezilla](https://docs.ovh.com/ca/fr/hosting/mutualise-guide-utilisation-filezilla/) puis copiez les fichiers contenus dans le dossier « **WordPress** » dans le « dossier racine » que vous avez défini sur votre hébergement lors de l'[étape 1](#step1) de ce guide.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.fr-ca.md]:
```suggestion
Une fois les fichiers décompressés dans votre dossier « **WordPress** », [connectez-vous en FTP à votre espace de stockage](https://docs.ovh.com/ca/fr/hosting/connexion-espace-stockage-ftp-hebergement-web/) à l'aide du [client FTP FileZilla](https://docs.ovh.com/ca/fr/hosting/mutualise-guide-utilisation-filezilla/) puis copiez les fichiers contenus dans le dossier « **WordPress** » dans le « dossier racine » que vous avez défini sur votre hébergement lors de l'[étape 1](#step1) de ce guide.

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.it-it.md [hunk 7]
```diff
@@ -6,199 +6,300 @@ section: CMS
...
+Se tutto è andato bene, visualizzi la pagina seguente:
 
-Per risolverlo, elimina o rinomina il file "index.html" che OVH ha salvato sul tuo hosting.
+![hosting](images/WPafterDB.png){.thumbnail}
 
-Se rinomini semplicemente il file, puoi riattivarlo in qualsiasi momento per utilizzarlo come pagina di attesa.
+Clicca su `Avvia l'installazione`{.action}.
 
-Altre informazioni utili: per essere presi in carico, i file del tuo sito devono trovarsi nella cartella "www".
+### 3.3 - Definisci l'accesso Amministratore al back office del tuo WordPress e la tua email di contatto
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.it-it.md]:
```suggestion
#### 3.3 - Definisci l'accesso Amministratore al back office del tuo WordPress e la tua email di contatto

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.it-it.md [hunk 8]
```diff
@@ -6,199 +6,300 @@ section: CMS
...
+
+## Per saperne di più <a name="go-further"></a>
+
+[Sito ufficiale WordPress](https://wordpress.org)
+
+Per prestazioni specializzate (referenziamento, sviluppo, ecc...), contatta i [partner OVHcloud](https://partner.ovhcloud.com/it/).
+
+Per usufruire di un supporto per l'utilizzo e la configurazione delle soluzioni OVHcloud, è possibile consultare le nostre soluzioni [offerte di supporto](https://www.ovhcloud.com/it/support-levels/).
+
+Contatta la nostra Community di utenti all'indirizzo <https://community.ovh.com>.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.it-it.md:305]:
```suggestion
Contatta la nostra Community di utenti all'indirizzo <https://community.ovh.com/en/>.
```

Comment (OVH-Team-Guides) [reply pages/web/hosting/cms_manual_installation_wordpress/guide.it-it.md:305]: Please check there is an /en/ link on all subs but FR ones

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md [hunk 9]
```diff
@@ -6,222 +6,300 @@ section: CMS
 order: 07
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md]: Please check all versions now contain a translation of "Tutorial - " inside the title
It seems to be missing here

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md [hunk 10]
```diff
@@ -6,222 +6,300 @@ section: CMS
...
+tar xvf latest.tar.gz
+```
 
-## Etap 2
-Zaopatrz się w dane do logowania do bazy danych  (pomoc w tym zakresie jest dostępna na początku przewodnika).
+- W katalogu głównym utworzony jest katalog "**wordpress**". Przenieś jego zawartość do podstawy katalogu głównego:
 
-Kliknij na "Zaczynajmy!", aby przejść do kolejnego etapu.
+```bash
+mv wordpress/*
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md]:
```suggestion
mv wordpress/* ./

```

Comment (OVH-Team-Guides) [reply pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md]: please check code-blocks are ok on all subs

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md [hunk 11]
```diff
@@ -6,222 +6,300 @@ section: CMS
...
-Ważne:  Dane do bazy danych nie są wysyłane automatycznie po zainstalowaniu hostingu. Aby je otrzymać, należy założyć bazę danych w panelu klienta.
+Wpisz swoją domenę na pasku wyszukiwania przeglądarki internetowej.
 
-Kliknij na "Wyślij", aby kontynuować.
+Jeśli pliki źródłowe WordPress zostały poprawnie umieszczone w katalogu głównym, pojawi się strona WordPress pozwalająca na wybór języka:
 
+![hosting](images/WPselectlangue.png){.thumbnail}
 
-- Etapy te kończą tworzenie połączenia między bazą danych i modułem WordPress. Wystarczy dokończyć instalację.
+Wybierz język strony i kliknij na 'Kontynuuj{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md]:
```suggestion
Wybierz język strony i kliknij na `Kontynuuj`{.action}.

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pt-pt.md [hunk 12]
```diff
@@ -6,226 +6,300 @@ section: CMS
...
+>
 
-## Etapa 2
-Uma vez abertas as pastas:
+#### 2.2 - Descomprimir os ficheiros de origem descarregados para uma nova pasta
 
+O ficheiro descarregado está no formato **compressado** (zippé). Crie uma pasta intitulada "**WordPress**" no seu computador e **descomprima** o conteúdo do ficheiro carregado no interior da pasta "**WordPress**".
 
-Na secção "Site local" encontrará os ficheiros necessários à instalação do CMS WordPress.
+Para isso, abra a pasta onde carregou o ficheiro comprimido, clique no ficheiro em questão e selecione "Extrair tudo...". ".
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.pt-pt.md]:
```suggestion
Para isso, abra a pasta onde carregou o ficheiro comprimido, clique no ficheiro em questão e selecione "Extrair tudo...". 

```

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pt-pt.md [hunk 13]
```diff
@@ -6,226 +6,300 @@ section: CMS
...
 
-Para iniciar a instalação de WordPress clique em "Instalar WordPress".
+- Título do sítio: introduza o título do seu site.
+- Identificador: defina o identificador Administrador do seu CMS.
+- Palavra-passe: defina uma palavra-passe para este identificador Administrador.
+- O seu endereço de e-mail: introduza um endereço de e-mail válido.
+- Privacidade: selecione esta opção para que os motores de busca referenciem o seu WordPress.
 
-![wordpress](images/3136.png){.thumbnail}
+Clique em `Installer WordPress`{.action} logo que as informações estiverem corretas.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manual_installation_wordpress/guide.pt-pt.md]: FR wording here

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md (hunk 9 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manual_installation_wordpress/guide.pl-pl.md]: done

File: pages/web/hosting/cms_manual_installation_wordpress/guide.pt-pt.md (hunk 13 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manual_installation_wordpress/guide.pt-pt.md]: done

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**